lib/Minify/Minify/Cache/W3TCDerived.php

```diff
@@ -85,8 +85,9 @@ public function isValid($id, $srcMtime)
     public function display($id)
     {
         $v = $this->fetch($id);
-        if (isset($v['content']))
-            echo $v['content'];
+		if ( isset( $v['content'] ) ) {
+			echo esc_html( $v['content'] );
```

Comment:
Check if content needs to get escaped.

Reply:
Switched to wp_kses with call to Util_Ui::get_allowed_html_for_wp_kses_from_content for allowed HTML tags